packages/twilio-run/src/serverless-api/utils.ts

```diff
@@ -35,6 +36,18 @@ export async function getFunctionServiceSid(
   if (username) {
     debug('Attempting to read serviceSid from a deployinfo file');
     const deployInfoCache = getDeployInfoCache(cwd);
+    if (
+      deployInfoCache &&
+      deployInfoCache[`${username}:${region}`] &&
+      deployInfoCache[username].serviceSid
```

Comment:
```suggestion
      deployInfoCache[`${username}:${region}`].serviceSid
```
Otherwise this if statement would fail.

Can you also make sure to add a fallback if `deployInfoCache[username]` is available but `deployInfoCache[`${username}:${region}`]` is not available?

Reply:
For fallback, I have kept the logic as is [here](https://github.com/twilio-labs/serverless-toolkit/pull/433/files#diff-3a6a3698899b9890e5faaef4af544dcf18f4bd8f68a1d7edf335fbedbf81a2d5R51-R62)

Would that suffice?

Reply:
Ah yeah that should work. It was collapased in the code review tab